paloma-testnet-12/genesis.json

```diff
@@ -205,9 +205,27 @@
+      }
```

Comment:
Shouldn't there be another one of these?

Reply:
no, it's the same smart contract code that is deployed across all evm chains. So we only have one ABI for the evm section and list all evm chains that we are on